portal/guides/create-gasless-nft-drop.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: How to create a gasless NFT drop with Next.js and thirdweb
+author: Avneesh Agarwal
+authorAvatar: "/assets/portal/authors/avneesh.jpg"
+authorTwitter: "https://twitter.com/avneesh0612"
+draft: false
+tags: [edition-drop, react, claim, gas]
+summary: "Create a gasless NFT drop with Next.js and thirdweb"
```

Comment:
this should be a better explainer and subtitle of what the guide would be about, my suggestion:

"Learn how to create an NFT Drop in which you pay the gas for your users, using OpenZeppeling, Next.js and thirdweb"